packages/core/bootstrap/src/lib/cache/metrics.ts

```diff
@@ -0,0 +1,39 @@
+import * as client from 'prom-client'
+import { MAXIMUM_MAX_AGE } from './index'
+
+enum CacheTypes {
+  Redis = 'redis',
+  Local = 'local',
+}
+
+export const cacheExecutionDurationSeconds = new client.Histogram({
```

Comment:
Consider changing this form seconds to milliseconds as it could be a better unit to measure duration here.

Also, let's make a style change and explicitly lowercase the metrics instances, so it's recognisable they are metrics(different than other vars) and we call them the same way we name them (e.g.,  `cache_execution_duration_seconds`).